content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration>
+```
+
+You may not want to run the commands above without also filtering the set of tests to run. Without a filter the command will run every test, and likely result in multiple hours of test execution! If you ever want to stop CTest, send an interrupt signal by selecting the terminal and pressing `CTRL + C` . The most commonly used suites are Smoke and Main, which have requirements to execute fast. The following ctest command filters to only tests in these two suites:
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration> -L "(SUITE_smoke|SUITE_main)"
+```
+
+The build configuration should be one you have already built locally. Running the Smoke and Main suites for the `profile` build configuration on a Linux machine should look like:
+
+```shell
+cd user/github/o3de/build/linux
+ctest -C profile -L "(SUITE_smoke|SUITE_main)"
+```
+
+After running CTest, results save to `.../<build_folder>/Testing/`.  If you prefer to see a summary of failures in your terminal, add the flag `--output-on-failure`.
+
+For more information on CTest usage, refer to its [online documentation](https://cmake.org/cmake/help/latest/manual/ctest.1.html).
+
+### Adding test modules to CTest
+
+CTest registers entire modules of test code. These modules typically contain multiple individual tests of the same feature. Adding new tests has three major steps, with a prerequisite:
+
+0. Add a build target for the production code which tests will target.
+1. Add a build target for tests which need to be compiled (not required for Python).
+2. Register your test module in a `CMakeLists.txt`.
+3. Add individual test functions into the test module.
+
+Specific steps vary for different types of tests, discussed in later sections.
+
+## GoogleTest
+
+Many tests are easiest to write in the same language as the production code they target. Since the majority of O3DE uses C++, the majority of its tests also use C++. O3DE's C++ tests are typically small [unit level](https://softwaretestingfundamentals.com/unit-testing/) tests of specific low-level functionality, which run extremely fast. O3DE uses [GoogleTest](https://github.com/google/googletest/blob/main/docs/index.md) to coordinate these tests, with an execution wrapper named AzTestRunner. CTest will call AzTestRunner to load the test library and the target library, and then execute any loaded GoogleTest tests.
+
+When building production C++ code for O3DE features, that code gets built into a library to later load into the engine. When writing C++ tests for this code, the test-code should be built into a separate library with a dependency on the production library. Building separate test libraries has the advantage of keeping production binaries lean and easy to ship, free of extraneous ```#if defined``` blocks for test code, and ensures we test the exact same interfaces that we ship.
```

Comment:
Suggestions to simplify sentences and make them flow better.
```suggestion
Production C++ code for O3DE features gets built into a library, which can later be loaded into the engine. When writing C++ tests for this code, the test-code should be built into a separate library with a dependency on the production library. Building separate test libraries has the advantage of keeping production binaries lean and easy to ship and free of extraneous ```#if defined``` blocks for test code. It also ensures we test the exact same interfaces that we ship.
```

Reply:
Incorporating Chanelle's suggestions with some other changes:

```suggestion
Production C++ code for O3DE features gets built into a library, which can later be loaded into the engine. When writing C++ tests for this code, the test-code should be built into a separate library with a dependency on the production library. Building separate test libraries has the advantage of keeping production binaries lean and easy to ship and free of extraneous ```#if defined``` blocks for test code. It also ensures test interfaces are identical to production interfaces.
```

The use of 2nd person in documentation ("we") should generally be avoided for a number of reasons, but primarily because the usage of this term implies the endorsement of the O3DF or O3DE project owners. This is why it's used only in _best practices_-style writing ("we recommend" i.e. "the O3DE project recommends") regarding neutral technical requirements that fit with efficient project usage.

Reply:
Accepted, though revised as interfaces for test libraries and production libraries are not identical.